src/containers/Header/index.jsx

```diff
@@ -21,15 +21,17 @@ const Header = (props) => {
         >
           <Img src={enterpriseLogo || EdxLogo} alt="" />
         </Link>
-        <Dropdown
-          title="edx@example.com"
-          menuItems={[
-            {
-              label: 'Logout',
-              href: '/',
-            },
-          ]}
-        />
+        {props.location.pathname.indexOf('/login') === -1 &&
+          <Dropdown
+            title="edx@example.com"
```

Comment:
We'll probably need to store the user's email address in the Redux store so that the `Dropdown` knows what email address to use here.